charts/dependency-track/values.yaml

```diff
@@ -74,6 +73,9 @@ tolerations: []
 affinity: {}
 
 env: []
+  ## Example: Configure JAVA_OPTIONS
```

Comment:
Note that JVM heap-size is calculated from limit: https://github.com/DependencyTrack/dependency-track/blob/master/src/main/docker/Dockerfile#L43